frontend/src/container/TraceDetail/index.tsx

```diff
@@ -48,6 +48,12 @@ function TraceDetail({ response }: TraceDetailProps): JSX.Element {
 		[response],
 	);
 
+	const traceStartTime = useMemo(() => response[0].startTimestampMillis, [
```

Comment:
will `startTimestampMillis` be always present in the response ?

Reply:
Yes

Reply:
and can the value of `startTimestampMillis` be ever undefined ?

Reply:
We have the default values if these values don't exist.

Reply:
yes if it can never be `undefined` we should make these mandatory props.

Reply:
gives us tighter type safety than optional props. can be easy to miss when reusing this component somewhere else.

Reply:
No, it's not possible unless backend returns 500.

Reply:
It's fine. For some reason if this fails, we have a fallback to old behavior

Reply:
we can remove the optional params and it will still fallback to the older behaviour